packages/keyring-eth-trezor/src/trezor-keyring.ts

```diff
@@ -116,14 +125,14 @@ export class TrezorKeyring extends EventEmitter {
     return this.bridge.model;
   }
 
-  init() {
+  async init(): Promise<void> {
```

Comment:
Was not `async` but since it's returning a `Promise` it makes sense to make it `async`.

Also, the `KeyringController` is already `await`ing those calls: https://github.com/MetaMask/core/blob/main/packages/keyring-controller/src/KeyringController.ts#L2199-L2201